test/helpers/index.js

```diff
@@ -0,0 +1,36 @@
+import express from 'express';
```

Comment:
Creating a test folder for all of the tests is a good move. I should probably merge my tests into this folder too so that everything can be found under this folder rather than scattered across the project. This should also fix nyc from counting test files in the code coverage results since we can just exclude this directory some how.